configure.ac

```diff
@@ -126,6 +126,17 @@ if test "$HAVE_LIBCL" -eq 1; then
     if test "x$GAWK" = "xno"; then
         AC_MSG_ERROR([gawk not found])
     fi
+
+    HAVE_CL_INTEL_H=1
+    AC_CHECK_HEADERS([CL/cl_intel.h], [HAVE_CL_INTEL_H=1], [HAVE_CL_INTEL_H=0])
+    if test "$HAVE_CL_INTEL_H" -eq 0; then
+        XCAM_WGET(
+            [https://raw.githubusercontent.com/intel/beignet/Release_v1.3/include/CL/cl_intel.h],
```

Comment:
then need check if file not exit or md5sum failed. then download the file.
and also need check wget return values. 0 => success, else report a error msg.